packages/web/src/pages/upload-page/forms/AttributionModalForm.module.css

```diff
@@ -0,0 +1,57 @@
+.row {
```

Comment:
I'm a fan of this but we could generalize it even further by putting these into their own file, like common classes